models/src_facebook_ads.yml

```diff
@@ -96,3 +98,27 @@ sources:
             description: A set of query string parameters which will replace or be appended to urls clicked from page post ads, message of the post, and canvas app install creatives only.
           - name: _fivetran_synced
             description: "{{ doc('_fivetran_synced') }}"
+          - name: page_link
+            description: Link for the page.
+          - name: template_page_link
+            description: URL destination of Facebook dynamic ads.
+          - name: asset_feed_spec_link_urls
+            description: Link to the asset feed spec
+          - name: object_story_link_data_child_attachments
+            description: Link of the object story child attachments
+          - name: object_story_link_data_caption
+            description: Link of the object story caption
+          - name: object_story_link_data_description
+            description: Link of the object story description
+          - name: object_story_link_data_link
+            description: Link of the object story link
+          - name: object_story_link_data_message
+            description: Link of the object story message
+          - name: template_app_link_spec_ios
+            description: Link of the object story spec for ios
+          - name: template_app_link_spec_ipad
+            description: Link of the template app spec for ipad
+          - name: template_app_link_spec_android
+            description: Link of the template app for android
+          - name: template_app_link_spec_iphone
+            description: Link of the template app for iphone
```

Comment:
I noticed the fields looked to have changed from what we previously documented in the yml. I didn't remove in case we meant to add them to the final model? If we don't use them we can remove the src and stg yml files.

Reply:
since those fields are selected in the staging model, i feel like we should document the min the yml? also i think `page_link` is added twice